build.xml

```diff
@@ -64,6 +64,7 @@
     <property name="build.classes" value="${build.dir}/classes"/>
     <property name="build.classes.main" value="${build.classes}/main" />
     <property name="javadoc.dir" value="${build.dir}/javadoc"/>
+    <property name="sstableloader-javadoc.dir" value="${build.dir}/sstableloader-javadoc"/>
```

Comment:
Move to ~660 where other sstableloader props are, rename to `sstableloader.javadoc.dir` for consistency.